quiche/quic/core/http/quic_spdy_session.h

```diff
@@ -53,7 +53,7 @@ class QUICHE_EXPORT Http3DebugVisitor {
 
   virtual ~Http3DebugVisitor();
 
-  // TODO(https://crbug.com/1062700): Remove default implementation of all
+  // TODO(crbug.com/1062700): Remove default implementation of all
```

Comment:
this was the only instances where the protocol was included -- i've dropped it for consistency